package.json

```diff
@@ -41,11 +42,12 @@
     "typescript": "^4.3.2"
   },
   "dependencies": {
+    "@types/prompts": "^2.0.11",
```

Comment:
do we need this dependency?
if so, shouldn't it be a dev dependency?

Reply:
Yes, `@types/prompts`  is for `prompts` , moved to devDependencies